bin/connector_config/Generate.hs

```diff
@@ -42,33 +42,42 @@ data Chapter
     | -- For now, we treat Eirika and Ephraim versions of the same map as the
       -- same location
       C Int
+    | T Int
+    | R Int
     | C5x
     | Endgame
     | Victory
+    | R10
     deriving (Show, Typeable)
 
 instance Bounded Chapter where
     minBound = Prologue
-    maxBound = Victory
+    maxBound = R10
 
 instance Enum Chapter where
     toEnum 0 = Prologue
     toEnum 6 = C5x
     toEnum 22 = Endgame
     toEnum 23 = Victory
+    toEnum 41 = R10
     toEnum i
         | i < 1 = error $ "invalid chapter index " ++ show i
         | i < 6 = C i
         | i < 22 = C (i - 1)
+        | i < 32 = T (i)
+        | i < 41 = R (i)
```

Comment:
This is wrong, we want index 24 to map to Tower 1, but as written this maps to Tower 24 (which doesn't exist).